garnett/reversion.py

```diff
@@ -0,0 +1,15 @@
+from garnett.fields import TranslatedField
+
+
+def patch_compare():
+    from reversion_compare.compare import CompareObjects as COBase
+
+    class CompareObjects(COBase):
+        def __init__(self, field, field_name, obj, version1, version2, is_reversed):
+            if isinstance(field, TranslatedField):
+                field_name = field.attname
+            super().__init__(field, field_name, obj, version1, version2, is_reversed)
+
+    import reversion_compare.compare
```

Comment:
Can we move this import to the top of the function?

Reply:
I think we want to import this as late as possible as we are monkey patching over it. I'll add a comment.